bucket/sqlitebrowser-nightly.json

```diff
@@ -0,0 +1,47 @@
+{
+    "version": "2022-06-18",
+    "description": "DB Browser for SQLite (DB4S) project (nightly build)",
+    "homepage": "https://github.com/sqlitebrowser/sqlitebrowser",
+    "license": "MPL-2.0|GPL-3.0-or-later",
+    "architecture": {
+        "64bit": {
+            "url": "https://nightlies.sqlitebrowser.org/win64/DB.Browser.for.SQLite-2022-06-18-win64.zip",
+            "hash": "c29c001e0bcd0e320abdb46e8d9dbb99fdca853a900274b10c28d88c1c1e02a3"
+        },
+        "32bit": {
+            "url": "https://nightlies.sqlitebrowser.org/win32/DB.Browser.for.SQLite-2022-06-18-win32.zip",
+            "hash": "edc4f1ded7b6cedacdd43a25cbd936ffc0ff4e2db54e4e49196fcc7609c2d0f0"
+        }
+    },
+    "extract_dir": "DB Browser for SQLite",
+    "bin": [
+        "DB Browser for SQLite.exe",
+        "DB Browser for SQLCipher.exe"
+    ],
```

Comment:
Do the binaries accept CLI args?

Reply:
They [do](https://github.com/sqlitebrowser/sqlitebrowser/wiki/Command-Line-Interface)!